julia-package/brainflow/test/csp.jl

```diff
@@ -0,0 +1,11 @@
+using BrainFlow
```

Comment:
in order to julia to run this test automatically you should add it to https://github.com/brainflow-dev/brainflow/blob/master/julia-package/brainflow/test/runtests.jl